type/__apt_source/manifest

```diff
@@ -35,7 +35,7 @@ if [ -f "$__object/parameter/arch" ]; then
 fi
 
 if [ -f "$__object/parameter/signed-by" ]; then
-   signed_by="${signed_by:${signed_by} }signed-by=$(sed -e ':a' -e '$!N' -e 's/\n/,/' -e 'ta' "$__object/parameter/signed-by")"
+   signed_by="signed-by=$(sed -e ':a' -e '$!N' -e 's/\n/,/' -e 'ta' "$__object/parameter/signed-by")"
```

Comment:
I love to see comments before funky sed/grep/awk oneliners, which aren't obvious on first sight. Usually these comments are written by me (many years ago) and I thank myself for writing them, because it helped me to understand what I meant with funky oneliner :smile:

Reply:
I would loved it too, but then figured out quite quickly, what it means - if you look closely, that `sed` magic isn't something I added. 🤣 

Being more serious, it's a fair point...